lib/file-utilities.js

```diff
@@ -270,18 +248,40 @@ function streamAllLogs(config, callback) {
     streamAllLogs(config)
   }, config.RESCAN_INTERVAL) // rescan for files every once in awhile
 }
-
-function gracefulShutdown(signal) {
+function processChunk(data) {
+  try {
+    data = JSON.parse(data)
```

Comment:
We should avoid mutating inputs whenever possible.  It makes it confusing for no reason (re-using the var provides little efficiency).  Here, just change the parameter name to `input` and declare a new var.  That way the error can return `input` and it's clear to the reader that the parse failed at that point.

```js
function processChunk(input) {

  let data
  try {
     data = JSON.parse(input)
  } catch (err) {
    return {t: Date.now(), l: input, f: 'systemd'}
  }
```